dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ProcessInstanceServiceImpl.java

```diff
@@ -335,6 +339,19 @@ public Result queryProcessInstanceList(User loginUser, long projectCode, long pr
         return result;
     }
 
+    /**
+     * check if the processInstance is finish
+     * @param processInstance processInstance
+     * @return result
+     */
+    private boolean isFinish(ProcessInstance processInstance){
+        if (processInstance.getState() == null) {
+            return false;
+        }
+        return !processInstance.getState().equals(WorkflowExecutionStatus.RUNNING_EXECUTION)
```

Comment:
@SbloodyS @caishunfeng I'm not sure about this, could u plz help take a look when available? Thanks : )
https://github.com/apache/dolphinscheduler/blob/2e4a9e6d5395730282a3f105aa815a4095a96433/dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/enums/WorkflowExecutionStatus.java#L29-L41

Reply:
has removed this new function, use the common `processInstance.getState().isFinished()`